### DISCUSSION TITLE
ERROR	[misconfig] Falling back to embedded checks	err="failed to download built-in policies: download error: OCI repository error: 1 error occurred:\n\t* GET https://ghcr.io/v2/aquasecurity/trivy-checks/manifests/1: TOOMANYREQUESTS: retry-after: 1.118302ms, allowed: 44000/minute\n\n"

### DISCUSSION THREAD

**Discussion by pswamicertinia** [aquasecurity/trivy#7870] [Bugs]
### Description

Getting this error from trivy-checks database which failed to download the builtin policies, similar error was fixed for vulnerability database error by providing one extra database "public.ecr.aws/aquasecurity/trivy-db". I am expecting the similar resolution for this error also If I am not wrong.
![image](https://github.com/user-attachments/assets/af7cd694-b570-4b11-be78-7cc01ace4c7a)

### Desired Behavior

It should download the latest database for misconfiguration scan. 

### Actual Behavior

Failing to download the the latest database for misconfiguration and throwing this error " ERROR	[misconfig] Falling back to embedded checks	err="failed to download built-in policies: download error: OCI repository error: 1 error occurred:\n\t* GET https://ghcr.io/v2/aquasecurity/trivy-checks/manifests/1: TOOMANYREQUESTS: retry-after: 1.118302ms, allowed: 44000/minute\n\n"

### Reproduction Steps

```bash
1. Run Trivy using Trivy go-binary 0.56.1
2. 
3.
...
```

### Target

Filesystem

### Scanner

Misconfiguration

### Output Format

Table

### Mode

None

### Debug Output

```bash
2024-11-04T14:11:54Z	ERROR	[misconfig] Falling back to embedded checks	err="failed to download built-in policies: download error: OCI repository error: 1 error occurred:\n\t* GET https://ghcr.io/v2/aquasecurity/trivy-checks/manifests/1: TOOMANYREQUESTS: retry-after: 1.118302ms, allowed: 44000/minute\n\n"
```

### Operating System

Linux

### Version

```bash
0.56.1
```

### Checklist

- [X] Run `trivy clean --all`
- [X] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by BSmith-APFM** [comment]
I started getting this error too.  The response on my call is:
`TOOMANYREQUESTS: retry-after: 1.13064ms, allowed: 44000/minute\n\n`
 seems to indicate there is a bandwidth error on ghcr.io where Aquasecurity/trivy is hosted on GitHub's infrastructure.  Can anyone confirm this is just a transient condition?

**Comment by simar7** [comment]
Please see https://github.com/aquasecurity/trivy/discussions/7699

**Comment by pswamicertinia** [comment]
@simar7 over #7699 I see that there are several public database resources available for both Vulnerability-DB and Java-DB, but none for the Trivy-checks DB, which is where I am encountering the actual error.

**Reply by simar7** [reply]
Sorry I didn't realize we didn't update that discussion for the checks bundle. I have done so now.

But in short, your scans are unaffected for misconfiguration scanning as Trivy embeds a copy of the checks within itself. You can see in your log output:

```[misconfig] Falling back to embedded checks```
